Dockerfile-default-rocm

```diff
@@ -1,40 +1,138 @@
 ARG BASE_IMAGE
 FROM ${BASE_IMAGE}
+#RUN sudo sysctl -w kernel.core_pattern="core-%h-%e-%s-%t-%u-%p"
+
+#RUN dpkg -S /opt/ompi
+RUN apt remove -y openmpi ucx
+#RUN mv /opt/ompi /opt/ompi.0 && mv /opt/ucx /opt/ucx.0
+RUN mv /opt/ompi /opt/ompi.0 
+#RUN ln -s /container/ompi /opt && ln -s /container/ucx /opt
+RUN ln -s /container/ompi /opt 
+#RUN ls /opt/ompi && exit 99
 
 RUN mkdir -p /var/run/sshd
+#RUN pip install torch torchvision torchaudio
```

Comment:
supposed to remove torch?

Reply:
Agreed, all commented out code should be removed.

Reply:
Commented out lines have been removed.